src/widgets/login-form/ui/LoginForm.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { Box, Grid, Link } from "@mui/material";
+import {
+  textFieldStyle,
+  firstTextFieldStyle,
+  loginButtonStyle,
+  linkStyle,
+  gridContainerProps,
+  gridItemProps,
+  rootStyle,
+} from "./style";
+import { CustomTextField } from "../../../shared/ui/CustomTextField";
+import { CustomButton } from "../../../shared/ui/CustomButton";
+import { PasswordField } from "../../../shared/ui/PasswordField/PasswordField";
```

Comment:
Во-первых, в этой ветке отсутствует компонент `PasswordField`. Не знаю, повлияет ли это на что-то, но в следующий раз лучше начинать делать виджет, когда все компоненты для него готовы.
Во-вторых, импорт скорее всего должен выглядеть так:
```suggestion
import { PasswordField } from "../../../shared/ui/PasswordField";
```